raiden/utils/ethereum_clients.py

```diff
@@ -24,6 +25,32 @@ def parse_geth_version(client_version: str) -> Optional[tuple]:
     return parse_version(matches.groups()[0])
 
 
+def support_check(
```

Comment:
Would be nice to have unit tests for this function.

Besides that: where did you find the `Version`  class? I cannot find it.

Reply:
It's tested via the unit tests in test_cli.py (https://github.com/raiden-network/raiden/pull/4612/files#diff-4f31d096585cf25811afbaf6fcc4fb7c)

> Besides that: where did you find the Version class? I cannot find it.

I went in the virtual env dependency of pkg_resources and found it in there. I wanted to see what does `parse_version` return. For the full path check how I import it in that file.